src/pages/umi/kit-adapters.md

```
@@ -0,0 +1,136 @@
+const kitInstruction = toKitInstruction(umiInstruction);
+```
+
+## Use Cases
+
+These adapters are particularly useful when:
+
+- You want to use Umi and Metaplex functionality together with `@solana/kit`
+- Building applications that need to interoperate between different parts of the Solana ecosystem
+- Integrating existing code that uses different type systems
+
+The adapters ensure type safety and handle the conversion details automatically, making it easy to work with both libraries in the same project.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)